### PR TITLE
Update Lightweight exporter and HO latency calculation logic

### PR DESCRIPTION
**Pull request by woojoong88**

### CODE REVIEW

File: pkg/exporter/exporter.go [hunk 1]
```diff
@@ -141,70 +133,33 @@ func exposeUEAdmRequests(mgr *manager.Manager) []prometheus.Counter {
...
-			listUELinkInfo = append(listUELinkInfo, tmp)
-		default:
-		}
-	}
-	return listUELinkInfo
+func initHOHistogram(mgr *manager.Manager) {
+	hoLatencyHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "ho_histogram",
+			Buckets: prometheus.LinearBuckets(0, 1000, 1000),
```
Comment (SeanCondon) [inline pkg/exporter/exporter.go]: 1000 buckets - that's a lot - I found the logarithmic buckets far more meaningful. You can cover 1ms to 2200ms in 20 buckets with a simple logarithmic scale like
```
Buckets:   prometheus.ExponentialBuckets(1e3, 1.5, 20),
```

File: pkg/exporter/exporter.go [hunk 2]
```diff
@@ -141,70 +133,33 @@ func exposeUEAdmRequests(mgr *manager.Manager) []prometheus.Counter {
...
+					Name: "ho_events",
+					ConstLabels: prometheus.Labels{
+						"timestamp": e.Timestamp.Format(time.StampNano),
+						"crnti":     e.Crnti,
+						"plmnid":    e.DestPlmnID,
+						"ecid":      e.DestECID,
+					},
+				})
+				eTime := float64(e.ElapsedTime)
+				tmp.Add(eTime)
```
Comment (SeanCondon) [inline pkg/exporter/exporter.go]: You don't seem to do anything with "tmp" - I don't get what the point in creating it was?

File: pkg/exporter/exporter.go (hunk 1 above)
Comment (woojoong88) [reply pkg/exporter/exporter.go]: Ok, that make sense. Let me change it.

File: pkg/exporter/exporter.go (hunk 2 above)
Comment (woojoong88) [reply pkg/exporter/exporter.go]: Here, I just used tmp, a Prometheus counter, to add elapsed time as a value. Nothing else.

Comment (SeanCondon) [reply pkg/exporter/exporter.go]: OK - i didn't see "tmp" being assigned to anything - so i thought it was just going to be discarded - I think I get it now - NewCounter creates it and adds it to the metrics it in one step?

Comment (woojoong88) [reply pkg/exporter/exporter.go]: Yes I think so. In my experience, the NewCounter is automatically registered, even if there is no `Registration` step.

**Pull request merged**